### ISSUE TITLE
Add screenShape

### ISSUE THREAD

**Issue by Daandelange** [Karma-Kusala/karmaMapper#3]
A rectangle that will always fit the screen dimensions.

**Comment by maxdee** [comment]
Hi Dan, thanks for input.
Its a bit of a niche feature, for now the best I could do is to snap to screen corners.
Going to add it now, it will be in the devel branch once I finish this refactoring blitz.
Dont hesitate to post more feedback!

**Comment by Daandelange** [comment]
Partially implemented but some functionality is missing, so leaving open.

**Comment by Daandelange** [comment]
@maxdee, yeah, snapping to screen corners is a good idea too, I'll keep that in mind. :)
But having a derived vertexShape* allows to bind the windowResizeEvent too.
